### DISCUSSION TITLE
Add Vic20 to vice Emulators

### DISCUSSION THREAD

**Discussion by garymeg** [slajerek/RetroDebugger#21] [Ideas]
Any chance of including vic20 to the debugger?

**Comment by slajerek** [comment]
I plan to add C128 and VIC20 someday soon, but I am rather thinking about adding Apple-II and Commodore Plus4 first.

**Reply by Dazdfa** [reply]
Would you also consider BBC Micro?

**Comment by Mythic6502** [comment]
Indeed, VIC-20 support would be really appreciated!

**Comment by phillipeaton** [comment]
Plus+1 for VIC version 👍

**Comment by Mythic6502** [comment]
Just a quick bump to see if the VIC20 might be in the pipeline
